.github/workflows/ci.yml

```diff
@@ -0,0 +1,409 @@
+name: Continuous Integration
+env:
+  DOCKER_IMAGE: usabillabv/php
+  DOCKER_BUILDKIT: 1
+  DOCKER_CLI_EXPERIMENTAL: enabled
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 3 * * 1,4'
```

Comment:
Does this mean the actions will run on this schedule regardless of any code changes? Super nice!

Reply:
Yeap. We still need to decide the frequency, but at least once per week, on a Friday, we'll have fresh images 🍋

Reply:
FYI we just decided to push this to Monday so if anything breaks on Friday it's not an issue for two and a half more days. If it breaks on Monday morning you can jump on it and fix it before it accumulates a weekend worth of warnings.